fed/barriers.py

```diff
@@ -366,7 +366,7 @@ def start_send_proxy(
     retry_policy=None,
     max_retries=None,
 ):
-    # Create RecevrProxyActor
+    # Create SendProxyActor
     global _SEND_PROXY_ACTOR
```

Comment:
I guess these could be refined as code cleaning stuff in other PRs.

Reply:
Sure